### ISSUE TITLE
virtiofs on DockerDesktop for Windows

### ISSUE THREAD

**Issue by dimadeush** [docker/for-win#13280]

**Comment by finite8** [comment]
This would make me so happy if we could have this implemented. I don't know if virtiofs would give the same benefits to windows as it does for MacOS, but it would be a game changer if it does.

**Comment by ra-mt** [comment]
Yes, VirtioFS feature in Docker would be much appreciated on Windows also!

**Comment by pocesar** [comment]
I think the most benefit would if WSL moved over to using virtfs instead of vp9, since it's massively slow. Docker wouldn't need to do much for getting the benefits of it

**Comment by nitzcard** [comment]
I need this feature too! :)

**Comment by illusive5943** [comment]
Any hope for this? Copying files to names volumes is painfully slow and bind mounts are unusable with large files such as LLMs.
